packages/cli/src/storyblok/storyblok-client.ts

```diff
@@ -10,6 +16,7 @@ type IsAuthenticatedFunc = () => Promise<boolean>
 type CreateFieldTypeFunc = (body: {
   name: string
   body?: unknown
+  options?: ManifestOption[]
```

Comment:
I've kept this property here although it's not accepted yet by the API during creation time.